### ISSUE TITLE
Blacklight8: Selected facet header text does not have sufficient contrast with background

### ISSUE THREAD

**Issue by maxkadel** [pulibrary/orangelight#4529]
### Expected behavior
When you select a facet to search by, the resulting facet header should be accessible.

### Actual behavior
The facet header has insufficient contrast with the background

On BL7 (passes contrast tests)
![image](https://github.com/user-attachments/assets/eecfb4ce-134c-4317-8766-8a8d824ff18b)

On BL8 (fails contrast tests)
![image](https://github.com/user-attachments/assets/1f107087-23b2-49dd-9cc5-c7ac7e6d555b)

### Steps to replicate
Either
a. Go to https://catalog-qa.princeton.edu/?f%5Blocation%5D%5B%5D=Mudd+Manuscript+Library
b. Go to the library homepage and select one of the facets to limit by

### Impact of this bug
Not accessible, cannot upgrade to BL8
